src/rest/QueryAnswerResponse.ts

```diff
@@ -0,0 +1,10 @@
+export interface IQueryAnswerResponse {
```

Comment:
What do you think of renaming the interface to `IQuestionAnswerResponse` and the file to a similar name?
I feel `query` is a little inconsistent with the rest of the naming.

Reply:
Good idea, I'll remove it.